well-architected/TOC.yml

```diff
@@ -269,6 +269,8 @@ items:
                 href: devops/checklist.md
               - name: Monitor cloud applications
                 items:
+                  - name: Monitoring Overview
```

Comment:
I added this because the monitoring page seems to be orphaned right now. Happy to use a different naming.